### PR TITLE
finished getSpeakerTime.ts

### PR DESCRIPTION
**Pull request by FangkuaikuaiOffical**

### CODE REVIEW

File: src/shared/getSpeakerTime.ts
Comment (weihanwang) [inline src/shared/getSpeakerTime.ts:1]: 1. I expect only backend would use it. so can move to `src/api`.
2. how about `parseSpeakerStats` instead? We may add more stats in addition to speaking time later on.
3. Can you add some test cases especially to cover corner cases such as empty string, a single line of transcript, two lines of transcript, incomplete transcript, malformed transcript.

File: src/shared/getSpeakerTime.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import fs from 'fs';
+
+interface Speaker {
```
Comment (weihanwang) [inline src/shared/getSpeakerTime.ts:3]: `SpeakerStats` instead

File: src/shared/getSpeakerTime.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  name: string;
+  totalSpeakingSeconds: number;
+}
+
+export function getSpeakerTime(filePath: string): Promise<Speaker[]> {
```
Comment (weihanwang) [inline src/shared/getSpeakerTime.ts:8]: 1. pass in the transcript string directly instead of a file. we aren't going to store transcripts as files.
2. without file reading it won't need to return Promise any more.
